allauth/account/views.py

```diff
@@ -28,6 +28,10 @@
 
 from .adapter import get_adapter
 
+import django
```

Comment:
Instead of checking for version numbers, alternatively you might want to:
```python
try:
    from django.contrib.auth import update_session_auth_hash
except ImportError:
    update_session_auth_hash = None
```

Then later on you can check for
```python
if (update_session_auth_hash is not None and
    not app_settings.LOGOUT_ON_PASSWORD_CHANGE):
```